### PR TITLE
Release/0.1.1

### PR DESCRIPTION
**Pull request by colmsnowplow**

### CODE REVIEW

File: examples/default/versions.tf [hunk 1]
```diff
@@ -0,0 +1,9 @@
+terraform {
+  required_version = ">= 1.0.0"
+}
+
+provider "azurerm" {
+  features {}
+}
+
+provider "azuread" {}
```
Comment (jbeemster) [inline examples/default/versions.tf:9]: Is this provider block needed?

**Review by jbeemster — COMMENTED** [review]
This all looks good - wondering if we need to bump the service module version up for consistency here as well?

File: examples/default/versions.tf (hunk 1 above)
Comment (colmsnowplow) [reply examples/default/versions.tf]: No it is not. Thanks for flagging. Removed.

**Comment by colmsnowplow** [comment]
@jbeemster thanks, addressed those points, if you review I'll then clean up for release. :)

**Review by jbeemster — COMMENTED** [review]
Looking good!  Only miss in the README - the module version bump for service + variable additions are not reflected there.

Would recommend always re-generating the docs when finalising a PR - not sure if its possible to do this automatically with a commit-hook instead?

**Comment by colmsnowplow** [comment]
OK, done - rebased and cleaned up for release too.

File: README.md [hunk 2]
```diff
@@ -145,14 +146,14 @@ module "collector_event_hub" {
 | Name | Description |
 |------|-------------|
 | <a name="output_nsg_id"></a> [nsg\_id](#output\_nsg\_id) | ID of the network security group attached to the Collector Server nodes |
+| <a name="output_user_data"></a> [user\_data](#output\_user\_data) | n/a |
```
Comment (jbeemster) [inline README.md:149]: I don't see any change to `outputs.tf` so wondering if this is an element of debug testing?

**Review by jbeemster — COMMENTED** [review]
Only one minor comment around the outputs change in the README but otherwise LGTM!

File: README.md (hunk 2 above)
Comment (colmsnowplow) [reply README.md]: Yup, an artefact of my debugging. Thanks for catching, fixed!

**Pull request merged**